spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md

```diff
@@ -126,28 +121,12 @@ Follow the below steps to enable accessing App Configuration with managed identi
     1. Create a TokenCredentialProvider and supply any valid TokenCredential and supply it via a Bean.
     1. Configure bootstrap.properties(or .yaml) in the Spring Boot application.
```

Comment:
We should have TokenCredentialProvider covered in a separate section instead of mixing with the managed identity. How about this?

3. Set the endpoint of your App Configuration store in the `bootstrap.properties` (or .yaml) file of your Spring Boot application. The system-assigned managed identity will be used. To use a user-assigned managed identity, additionally set the client id of your user-assigned managed identity, for example
```properties
spring.cloud.azure.appconfiguration.stores[0].endpoint=[config-store-endpoint]
spring.cloud.azure.appconfiguration.managed-identity.client-id=[client-id]
```

Reply:
Then we can have another section for TokenCredentialProvider. Something like

### Use Azure Active Directory credentials to access App Configuration and Key Vault.